api/v1beta1/verticadb_types.go

```diff
@@ -480,7 +489,7 @@ type Subcluster struct {
 	ServiceType corev1.ServiceType `json:"serviceType,omitempty"`
```

Comment:
Can we add a selector for this one?  We should select between ClusterIP, NodePort, and LoadBalancer